unofficial_flocker_tools/sample_files.py

```diff
@@ -9,3 +9,6 @@ def main():
         resource = resource_filename("unofficial_flocker_tools", "samples/" + filename)
         shutil.copyfile(resource, filename)
         print filename
+
+if __name__ == "__main__":
+    main()
```

Comment:
This is not necessary - this script can/should not be run directly, please remove it.